app/src/main/java/com/babylon/wallet/android/data/gateway/model/TokensAndLsusPricesResponse.kt

```diff
@@ -44,15 +45,15 @@ data class TokensAndLsusPricesResponse(
             val lsus = this.lsus.map { lsuPrice ->
                 TokenPriceEntity(
                     resourceAddress = ResourceAddress.init(lsuPrice.resourceAddress),
-                    price = lsuPrice.usdPrice.toDecimal192(),
+                    price = lsuPrice.usdPrice.toDecimal192OrNull().orZero(),
```

Comment:
from what I understood in the slack thread, this is a temp solution?
If yes then add a TODO with some comment there?

Reply:
Not necessarily, this fixes the current issue of not displaying the amounts at all because of the exception being thrown by this line, now this won't throw the exception anymore and the fallback to zero is good to have permanently. As per the actual fix for the prices with more than 18 decimals, most likely it will either require API changes or Sargon changes and a version bump in the wallet, so I'm not sure what's left TODO here.

Reply:
makes sense, thanks

Reply:
It was mentioned that the API was fixed.